### DISCUSSION TITLE
Suggestion: Nextcloud

### DISCUSSION THREAD

**Discussion by kszere** [tteck/Proxmox#353] [Ideas]
Nextcloud software is great.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/4

**Reply by tteck** [reply]
@kszere, NextCloudPi Done!

**Reply by sking0379** [reply]
Dumb question here, I installed Nextcloud Alpine, but I am not able to login, is there a default username and password..? The URL is not allowing me to create an account.

**Reply by tteck** [reply]
![Screenshot 2023-11-30 10 50 31 AM](https://github.com/tteck/Proxmox/assets/21342670/20f9f98c-df9b-4bdd-8029-fffa58c45921)
